server/routerlicious/packages/services/src/mongoExceptionRetryRules/mongoError/index.ts

```diff
@@ -129,8 +140,7 @@ class ServiceUnavailableRule implements IMongoExceptionRetryRule {
 class TopologyDestroyed implements IMongoExceptionRetryRule {
     private static readonly message = "Topology was destroyed";
     match(error: any): boolean {
-        return error.code === 16
-            && error.message
+        return error.message
```

Comment:
Was error code 16 not appropriate here?

Reply:
Yes, again a copy paste error, the sample exception is :
> {"name":"MongoError","message":"Topology was destroyed","stack":"MongoError: Topology was destroyed\n    at initializeCursor (\/usr\/src\/server\/node_modules\/mongodb-core\/lib\/cursor.js:596:25)\n    at...